server.py

```diff
@@ -6,5 +7,6 @@
 if __name__ == '__main__':
     # Startup
     logging.basicConfig(level=settings.LOGGING_LEVEL, format=settings.LOGGING_FORMAT)
+    logging.info("Starting server: version='{}'".format(__version__))
```

Comment:
@necrophonic can you approve this based on my comment [here](https://github.com/ONSdigital/sdx-sequence/pull/11) - the reasons are the same